src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <v-container id="name-options-container">
+    <h4 class="name-options-info mb-3">You can correct the company name in one of the following ways:</h4>
+    <v-expansion-panels v-model="panel">
+      <v-expansion-panel
+        v-for="(item,i) in displayedOptions"
+        :key="i"
+      >
+        <v-expansion-panel-header class="name-options-header">{{item.description}}</v-expansion-panel-header>
+        <v-expansion-panel-content class="name-options-content">
+          <v-container>
+            <component :is="item.component" :key="item.id" />
+          </v-container>
+        </v-expansion-panel-content>
+      </v-expansion-panel>
+    </v-expansion-panels>
+
+    <div class="action-btns my-3">
+      <v-fade-transition hide-on-leave>
+        <v-btn id="done-btn" large color="primary" @click="emitSave()"><span>Done</span></v-btn>
+      </v-fade-transition>
+
+      <v-btn id="cancel-btn" large @click="emitCancel"><span>Cancel</span></v-btn>
+    </div>
+  </v-container>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+
+// Interfaces
+import { CorrectNameOptionIF } from '@/interfaces'
+
+/**
+ * Operation:
+ * 1. To initialize this component option list, pass in an array of id's of the components you want to display.
+ *    ie..[''correct-new-nr', 'correct-name', 'correct-name-to-number']
+ * 2. If this options list is only passed one value the option panel will be open by default.
+ * 3. The parent component will have to watch for the 'save' and 'cancel' events and handle them accordingly.
+ */
+@Component({})
+export default class CorrectNameOptions extends Vue {
+  /** The options to display */
+  @Prop() private correctionNameChoices: Array<string>
+
+  // local properties
+  private displayedOptions: Array<CorrectNameOptionIF> = []
+  private panel = null as number
+  private correctionNameOptions: Array<CorrectNameOptionIF> = [
+    {
+      id: 'correct-name',
```

Comment:
Very neat.
I suggest you use an enum for the ids, so the caller and callee are in sync.

Reply:
Great idea! I will update this in the next commit